examples/tree_2d_dgsem/elixir_euler_blast_wave_neuralnetwork_rayhesthaven.jl

```diff
@@ -1,9 +1,9 @@
 using Downloads: download
 using Flux
-using NNlib
+using Random
```

Comment:
Why do we need `Random` here?

Reply:
Required by the new `Dropout` type in Flux.jl. If missing, we get an error on the line where the BSON file is loaded à la
```julia
julia> trixi_include("examples/tree_2d_dgsem/elixir_euler_blast_wave_neuralnetwork_rayhesthaven.jl", initial_refinement_level = 4, maxiters=50)
ERROR: LoadError: UndefVarError: Random not defined
Stacktrace:
  [1] (::BSON.var"#31#32")(m::Module, f::String)
    @ BSON ~/.julia/packages/BSON/rOaki/src/extensions.jl:21
[...]
 [41] load(x::String, init::Module)
    @ BSON ~/.julia/packages/BSON/rOaki/src/read.jl:108
 [42] top-level scope
    @ /mnt/ssd/home/mschlott/hackathon/bump-compat/examples/tree_2d_dgsem/elixir_euler_blast_wave_neuralnetwork_rayhesthaven.jl:6
 [43] include
    @ ./Base.jl:419 [inlined]
 [44] #trixi_include#1079
    @ /mnt/ssd/home/mschlott/hackathon/bump-compat/src/auxiliary/special_elixirs.jl:37 [inlined]
 [45] #trixi_include#1081
    @ /mnt/ssd/home/mschlott/hackathon/bump-compat/src/auxiliary/special_elixirs.jl:40 [inlined]
 [46] top-level scope
    @ REPL[2]:1
in expression starting at /mnt/ssd/home/mschlott/hackathon/bump-compat/examples/tree_2d_dgsem/elixir_euler_blast_wave_neuralnetwork_rayhesthaven.jl:6
```